maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/CheckoutProjectFromScm.java

```diff
@@ -270,13 +270,19 @@ public ReleaseResult simulate( ReleaseDescriptor releaseDescriptor, ReleaseEnvir
     {
         ReleaseResult result = new ReleaseResult();
 
+        MavenProject rootProject = ReleaseUtil.getRootProject( reactorProjects );
+        File checkoutDirectory =
+                        FileUtils.resolveFile( rootProject.getBasedir(), releaseDescriptor.getCheckoutDirectory() );
+
         if ( releaseDescriptor.isLocalCheckout() )
         {
-            logInfo( result, "This would be a LOCAL check out to perform the release ..." );
+            logInfo( result,
+                     "This would be a LOCAL check out to perform the release from " + checkoutDirectory + "..." );
```

Comment:
I prefer a strong on `local` rather than shouting it out load in upper case.

Reply:
good idea

Reply:
done